tracing-opentelemetry/README.md

```diff
@@ -0,0 +1,70 @@
+# Tracing OpenTelemetry
+
+[![Crates.io: tracing-opentelemetry](https://img.shields.io/crates/v/tracing-opentelemetry.svg)](https://crates.io/crates/tracing-opentelemetry)
+[![Documentation](https://docs.rs/tracing-opentelemetry/badge.svg)](https://docs.rs/tracing-opentelemetry)
+[![License: MIT](https://img.shields.io/badge/license-MIT-blue.svg)](LICENSE.txt)
+
+An OpenTelemetry layer for the [tracing] library.
+
+[tracing]: https://github.com/tokio-rs/tracing
+
+## Tracers
+
+Currently supports the Jaeger exporter via [opentelemetry-jaeger] and Zipkin
+via [opentelemetry-zipkin].
+
+[opentelemetry-jaeger]: https://github.com/open-telemetry/opentelemetry-rust/tree/master/opentelemetry-jaeger
+[opentelemetry-zipkin]: https://github.com/open-telemetry/opentelemetry-rust/tree/master/opentelemetry-zipkin
+
+Examples
+--------
+
+### Basic Usage
+
+```rust
+#[macro_use]
+extern crate tracing;
```

Comment:
Examples in the `tracing` repo _typically_ import `tracing`'s macro using the 2018 edition-stabilized `use tracing::{span, error}` syntax. Can we do that here as well?

Reply:
good point, will do 👍